Makefile

```diff
@@ -0,0 +1,12 @@
+all: test
```

Comment:
Not sure about adding a line of code _just_ for users to be able to write `make` instead of `make test` mak